Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1426,10 +1426,16 @@ extension BrowserViewController {
 
 extension BrowserViewController: URLBarDelegate {
     func showTabTray() {
+        guard !tabManager.tabsForCurrentMode.isEmpty else {
```

Comment:
This is a double negative. Just use an `if`:

```suggestion
        if tabManager.tabsForCurrentMode.isEmpty {
```